flatland_server/test/debug_visualization_test.cpp

```diff
@@ -100,19 +100,8 @@ TEST(DebugVizTest, testBodyToMarkersPolygon) {
   ASSERT_NEAR(markers.markers[0].pose.orientation.w, 0.70710678118, 1e-5);
 
   // Check the marker shape
-  ASSERT_EQ(markers.markers[0].type, markers.markers[0].LINE_STRIP);
-  ASSERT_EQ(markers.markers[0].points.size(),
-            5);  // box as line strip: 0, 1, 2, 3, 0
-  ASSERT_NEAR(markers.markers[0].points[0].x, -1.0, 1e-5);
```

Comment:
Why are you removing these asserts?
Don't remove tests without providing at least equivalent tests